docs/how-to-guides/how-to-speed-up-value-with-parallel.md

```diff
@@ -0,0 +1,197 @@
+---
+title: How-to speed up data valuation with parallelization
+alias:
+    name: speed-up-value-with-parallel
+---
+
+Real world data valuation problems often comprise more complex and larger
+models and/or datasets that are orders of magnitude larger and complex
+than the ones we see in the literature.
+
+In this guide we will show you how to speed up data valuation algorithms
+by leveraging parallelization either locally (within a single machine)
+using threads or processes or remotely (across multiple machines)
+using [Ray](https://ray.io) or [Dask](https://docs.dask.org/en/stable/).
+
+The former works out of the box but for the latter you will need to install
+additional dependencies (see [Extras][installation-extras])
+and to provide a running cluster.
+
+!!! tip "Other guides"
+
+    For scaling-up influence function algorithms with parallelization
+    see [[scale-up-if-with-parallel]].
+
+    For alternative ways to speed up data valuation
+    see [[speed-up-value-with-caching]].
+
+!!! warning "Scaling dataset size"
+
+    The entire dataset needs to fit in the memory of each and every worker.
+    If your dataset doesn't fit in memory, then you should consider
+    using a model that can be fitted incrementally with a dataset
+    that can be loaded partially.
+
+## Set up the dataset, model and method
+
+For the rest of the guide we will use the following dataset and model:
+
+```python
+from sklearn.datasets import fetch_covtype
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import MinMaxScaler, Normalizer
+from sklearn.svm import LinearSVC
+
+data = fetch_covtype()
+model = make_pipeline(MinMaxScaler(), Normalizer(), LinearSVC())
+```
+
+And the following scorer, utility and data valuation method:
+
+```python
+from pydvl.valuation import (
+    Dataset,
+    DataShapleyValuation,
+    ModelUtility,
+    PermutationSampler,
+    MaxUpdates,
+    Scorer,
+)
+
+scorer = Scorer("accuracy")
+utility = ModelUtility(model, scorer)
+training_set, test_set = Dataset.from_sklearn(data, random_state=16)
+valuation = DataShapleyValuation(
+    utility,
+    sampler=PermutationSampler(batch_size=10, seed=16),
+    is_done=MaxUpdates(100)
+)
+```
+
+## Parallelization
```

Comment:
This section is very good but it is explaining background instead of showing how to get stuff done. I would keep it in this document but switch the order of "Local Parallelization" and "Parallelization" and rename "Parallelization" to something like "Understanding the pattern" or "Understanding what happened". That way people first see the code example they can copy and paste and then can decide on their own if they want to continue reading.